www/source/stylesheets/vendor/foundation/components/_button-group.scss

```diff
@@ -84,7 +84,7 @@ $buttongroup-expand-max: 6 !default;
         &:first-child:nth-last-child(#{$i}) {
           &, &:first-child:nth-last-child(#{$i}) ~ #{$selector} {
             display: inline-block;
-            @if #{$buttongroup-spacing} == '0' {
+            @if '#{$buttongroup-spacing} == "0"' {
```

Comment:
This was a warning that kept appearing when I started middleman. The fix here was mistakenly checked in with all this other code. I am happy to leave it or remove it.